dashboard/test/lib/services/lti_test.rb

```diff
@@ -277,23 +277,29 @@ class Services::LtiTest < ActiveSupport::TestCase
   end
 
   test 'should add or remove student users when syncing a section' do
-    lti_integration = create :lti_integration
-    lti_course = create :lti_course, lti_integration: lti_integration
-    lti_section = create :lti_section, lti_course: lti_course
+    auth_id = "#{@lti_integration[:issuer]}|#{@lti_integration[:client_id]}|user-id-1"
+    user = build :teacher
+    user.authentication_options << build(:lti_authentication_option, user: user, authentication_id: auth_id)
+    user.save
```

Comment:
```suggestion
    user = create :teacher
    create :lti_authentication_option, user: user, authentication_id: auth_id
```

Reply:
Theres a [trait](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/test/factories/factories.rb#L447) you can use too, I think just `user = create :teacher, :with_lti_auth`